src/servers/FtpServer.cc

```diff
@@ -1126,10 +1126,13 @@ Ftp::Server::writeErrorReply(const HttpReply *reply, const int scode)
     }
 #endif
 
-    Must(reply);
-    const char *reason = reply->header.has(Http::HdrType::FTP_REASON) ?
-                         reply->header.getStr(Http::HdrType::FTP_REASON):
-                         reply->sline.reason();
+    const char *reason;
+    if (reply)
+        reason = reply->header.has(Http::HdrType::FTP_REASON) ?
+                 reply->header.getStr(Http::HdrType::FTP_REASON):
+                 reply->sline.reason();
+    else
+        reason = "Lost Error";
```

Comment:
I suggest making this the initial value. This will simplify the code slightly and address concerns that `reason` might be left uninitialized during future code changes.

```
const char *reason = "Lost Error"; // refined below if possible
```